MREUnityRuntime/MREUnityRuntimeLib/Core/User.cs

```diff
@@ -23,10 +23,13 @@ internal class User : MixedRealityExtensionObject, IUser
 
 		public UInt32 Groups { get; internal set; } = 1;
 
-		internal void Initialize(IHostAppUser hostAppUser, Guid userId, MixedRealityExtensionApp app)
+		public Guid InstancedUserId { get; private set; }
+
+		internal void Initialize(IHostAppUser hostAppUser, Guid userId, Guid instancedUserId, MixedRealityExtensionApp app)
 		{
 			HostAppUser = hostAppUser;
 			base.Initialize(userId, app);
+			InstancedUserId = instancedUserId;
```

Comment:
>InstancedUserId [](start = 3, length = 15)

maybe rename instancedUserID to EphemeralUserId throughout the code?

Reply:
I agree in keeping the ephemeral naming convention consistent here as well.